amida-tech/greyscale
# Feature/inba 990 jf trim tables

#### What does this PR do?
Removes various tables, controllers, tests, functions and files that are not needed by our current product, reducing the complexity and "red herrings" of Greyscale. 

#### Related JIRA tickets:

#### Did the user.integration, discussion.integration and task.integration tests pass? Test can be run with:
#### node_modules/.bin/mocha test/user.integration.js test/discussion.integration.js test/task.integration.js --bail -t 10000

`users.integration.js` works but the other two will not and are unlikely to have ever did. This is because they do not factor in the importance of creating the project first, then the product. Because this is a reduction in code, it did not touch upon these particular areas enough to merit fixing tests just yet. 

#### How should this be manually tested?

Go into your database instances. Completely wipe out your Indaba database and recreate it blank. Then in `amida-auth-microservice`, wipe out the users _other_ than Admin. If you set your auth service admin's password with `AUTH_SERVICE_SEED_ADMIN_PASSWORD`, be sure to set that in Greyscale as well.

Under `greyscale\backend`, run `yarn seed`. Then run `yarn start`. In another terminal, run `yarn test-seed`.

Check that 29 tables are generated in the `develop` or `testorg` namespace for Indaba's database. 

Regression test Indaba entirely. Record any bugs. 

#### Background/Context

#### Screenshots (if appropriate):
